src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/RedundantModifierCheck.java

```diff
@@ -134,6 +134,13 @@
  *   ProtectedInnerClass pc = new ProtectedInnerClass();
  * }
  * </pre>
+ * <p>
+ * Starting from Java 17, all floating point operations remains consistently strict,
```

Comment:
grammar note - it looks like it should be "remain", not "remains"

Reply:
Done